engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/docker-tag-repositories.factory.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.docker-tags.factory:DockerTagRepositories
+ *
+ * @requires BastionResource
+ *
+ * @description
+ *   Provides a BastionResource for Docker Tag Repositories
+ */
+angular.module('Bastion.docker-tags').factory('DockerTagRepositories',
```

Comment:
I think we don't need a whole new file for this  but let me know if you already tried the below:

I believe the DockerTag factory can be expanded to add the repositories endpoint and that would be invoked with `DockerTag.getRepositories` or something like that. You can refer to another file that makes me think it's possible:  https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host-bulk-action.factory.js

BTW - does this autocomplete stuff in here actually work?

Reply:
Looks like autocomplete is broken here now.  I will investigate if autocomplete can be fixed without too much trouble since we're filtering on repositories in the docker tags controller.  If not, I'll revert to editing the repositories controller like I was doing before.  It isn't worth getting too caught up on this.

Reply:
Edit: looks like autocomplete was never there to begin with, but the filter was indeed broken.  The new repo-centric table was missing it, I'm going to add it in my next commit.